waku/v2/node/wakunode2.nim

```diff
@@ -359,6 +359,20 @@ proc info*(node: WakuNode): WakuInfo =
   let wakuInfo = WakuInfo(listenStr: listenStr)
   return wakuInfo
 
+proc mountLightRelay*(node: WakuNode) = 
```

Comment:
Actually, can this just be mountRelay? Then have as an argument if we want to subscribe to default topic or not. Might be cleaner? So you have

mountRelay(sub=false)
mountRelay(sub=default)
mountRelay(sub=shard1, shard2, shard3...)
etc

Reply:
Yes, this makes sense. One extra layer of complexity here is that we want to (are forced to) mount relay on the switch without actually having the _capability_ to relay messages, but this works well in this model as just another argument.